src/css/components/_alert.scss

```diff
@@ -91,6 +91,9 @@
     height: calc((#{$master-height} - 75px) / 2);
     width: 50%;
     text-align: center;
+    p {
+        overflow: hidden;
```

Comment:
Still seeing the overflow happening horizontally.
![Alert Screenshot](https://user-images.githubusercontent.com/10703941/129607760-c5e5cfc5-7deb-4f94-aa53-79419d0ea0f7.png)

Might have to limit the size of the container